### ISSUE TITLE
Parameter "_AMT_CG_rt4" wasn't equiped with correct reference

### ISSUE THREAD

**Issue by GoFroggyRun** [open-source-economics/Tax-Calculator#519]
The `_AMT_CG_rt4` parameter in the `current_law_policy.json` file does not have the correct reference and name in the way that the parameter has nothing to do with capital gain rates (which only have three groups). This parameter of value 0.25 (25%) is likely to be the 25 percent described in the section "Netting capital gains and losses" on P640 in U.S. Master Tax Guide, 98th Ed. The exact source of the parameter will be further investigated.

**Comment by martinholmer** [comment]
@GoFroggyRun said on 06-Jan-2016:
> The _AMT_CG_rt4 parameter in the current_law_policy.json file does not have the correct reference and name in the way that the parameter has nothing to do with capital gain rates (which only have three groups). This parameter of value 0.25 (25%) is likely to be the 25 percent described in the section "Netting capital gains and losses" on P640 in U.S. Master Tax Guide, 98th Ed. The exact source of the parameter will be further investigated.

If this is a mistake, can you prepare a pull request that fixes the mistake?  I guess that pull request would involve some edits of the ```current_law_policy.json``` file, is that correct?

@MattHJensen @feenberg

**Comment by GoFroggyRun** [comment]
@martinholmer:
>
If this is a mistake, can you prepare a pull request that fixes the mistake? 

Yes. I believe the name of that parameter wasn't properly assigned. I'm, however, also not sure what's the best name for it. This 0.25 (25%) should be the exact same amount appear [hear](https://github.com/open-source-economics/Tax-Calculator/blob/master/taxcalc/functions.py#L668) on capital gain, where the capital gain 25% is being hard-coded. 
>
I guess that pull request would involve some edits of the `current_law_policy.json` file, is that correct?

And yes, if an appropriate name were found, I'll edit and incorporate in the `current_law_policy.json`.

**Comment by martinholmer** [comment]
@MattHJensen and Dan @feenberg,
Sean and I are confused by the the ```_AMT_CG_rt4``` policy parameter in the ```current_law_policy.json``` file.  The description of this parameter says this:
```
"long_name": "Long term capital gain and qualified dividend rate 4",
"description": "The capital gain and qualified dividend (stacked on top of regular income) above threshold 2 are taxed at this rate.",
```

But that is exactly the same description as for ```_AMT_CG_rt3```, which says:
```
"long_name": "Long term capital gain and qualified dividend rate 3",
"description": "The capital gain and qualified dividend (stacked on top of regular income) above threshold 2 are taxed at this rate.",
```

And our confusion is deepened when we look at the functions.py file to see where ```_AMT_CG_rt4``` is being used, which is **only** in the last line of this code fragment:
```
    # Capital Gain for AMT
    _tamt2 = 0.
    _amt5pc = 0.0
    _line45 = max(0., AMT_CG_thd1[MARS - 1] - c24520)
    _line46 = min(_alminc, c62720)
    _line47 = min(_line45, _line46)
    _line48 = min(_alminc, c62720) - _line47
    _amt15pc = min(_line48, max(0., AMT_CG_thd2[MARS - 1] - c24520 - _line45))
    if _ngamty != (_amt15pc + _line47):
        _amt20pc = _line46 - _amt15pc - _line47
    else:
        _amt20pc = 0.
    if c62740 != 0:
        _amt25pc = max(0, _alminc - _ngamty - _line46)
    else:
        _amt25pc = 0.
    c62747 = AMT_CG_rt1 * _amt5pc
    c62755 = AMT_CG_rt2 * _amt15pc
    c62760 = AMT_CG_rt3 * _amt20pc
    c62770 = AMT_CG_rt4 * _amt25pc
```
It looks to us that the ```62770``` variable has not been part of the of the PUF or the tax law for some years (at least this is what the spreadsheet of e-codes seems to suggest).  And it is not part of the IRS 2008 PUF file.

Did there used to be, in years before 2008, higher AMT tax rates for capital gains?  That's what it looks like to us, but we are not that well versed in tax law history.  If our hunch is correct (that ```_AMT_CG_rt4``` has not been part of the tax code, at least, since 2013), why is ```_AMT_CG_rt4``` in the ```current_law_policy.json``` file?  And, if there is a reason for its inclusion in the ```current_law_policy.json``` file, why is its description so confusing?

**Comment by GoFroggyRun** [comment]
@martinholmer 
I think including `_AMT_CG_rt4` in `current_law_policy.json` file is my mistake in #438. 
Previously, before PR #438 being merged, we have 
```
c62770 = 0.25 * _amt25pc,
```
and then the `0.25` was parameterized as `_AMT_CG_rt4`, which, now, is considered inappropriate. The reason why I did so might be that the name of the variable, `_amt25pc`, somehow confused me.

**Comment by martinholmer** [comment]
Sean said:
> I think including _AMT_CG_rt4 in current_law_policy.json file is my mistake in #438 [merged into the master branch by me on 05-Nov-2015].
> Previously, before PR #438 being merged, we have
> ```c62770 = 0.25 * _amt25pc```
> and then the 0.25 was parameterized as _AMT_CG_rt4, which, now, is considered inappropriate. The reason why I did so might be that the name of the variable, _amt25pc, somehow confused me.

@MattHJensen and Dan @feenberg,
So you can see that our confusion is long-standing and still remains.  Sean and I can now add to our list of questions: what is ```_amt25pc```?  This variable name seems to suggest that some part of AMT taxable income is taxed at 25%.  Has this been part of current-law policy since 2013?   As you can see, we need some help on this issue.

**Comment by feenberg** [comment]
On Sun, 14 Feb 2016, Martin Holmer wrote:

> 
> Sean said:
>
>       I think including _AMT_CG_rt4 in current_law_policy.json file is my
>       mistake in #438 [merged into the master branch by me on 05-Nov-2015].
>       Previously, before PR #438 being merged, we have
>       c62770 = 0.25 * _amt25pc
>       and then the 0.25 was parameterized as _AMT_CG_rt4, which, now, is
>       considered inappropriate. The reason why I did so might be that the name
>       of the variable, _amt25pc, somehow confused me.
> 
> @MattHJensen and Dan @feenberg,
> So you can see that our confusion is long-standing and still remains. Sean and I can
> now add to our list of questions: what is _amt25pc? This variable name seems to
> suggest that some part of AMT taxable income is taxed at 25%. Has this been part of
> current-law policy since 2013? As you can see, we need some help on this issue.

I think there is some cg taxed at 25% but not available in the PUF. I'll 
check this afternoon.

dan

**Comment by martinholmer** [comment]
dan @feenberg said:
> I think there is some cg taxed at 25% but not available in the PUF. I'll check this afternoon.

Thanks for looking into this, Dan.

**Comment by feenberg** [comment]
As far as I can tell the 25% rate applies to "Unrecaptured Schedule E 
Gain" and isn't something that would be in the PUF, but which is 
significant in the Insole files, so we should continue to carry it.

dan

**Comment by martinholmer** [comment]
Dan @feenberg said:
> As far as I can tell the 25% rate applies to "Unrecaptured Schedule E Gain" and isn't something that would be in the PUF, but which is significant in the Insole files, so we should continue to carry it.

Dan,  Thanks for looking into this matter.

So, maybe we should undo the confusion that Sean and I experienced months ago when we changed the  hardwired 0.25 tax rate to a Policy parameter in ```current_law_policy.json```.   I notice that the parameter we added is not referenced by TaxBrain, so I'll fix this up later this afternoon.

@MattHJensen @GoFroggyRun

**Comment by GoFroggyRun** [comment]
This issue has been solved in #599 . Thanks @martinholmer .
